### ISSUE TITLE
pydantic error

### ISSUE THREAD

**Issue by insukjoung** [OpenFreeEnergy/openfe#177]
Some of the class variables have an issue.

```python
from openfe.setup.methods.openmm.equil_rbfe_methods import (
    IntegratorSettings, SimulationSettings
)
from openff.units import unit

integrator_settings=IntegratorSettings(
    timestep = 2 * unit.femtosecond,
    temperature = 298.15 * unit.kelvin,
    collision_rate = 1 / unit.picosecond,
    n_steps = 1000,
    reassign_velocities=True,
    splitting='V R O R V',
    n_restart_attempts=20,
    constraint_tolerance=1e-06,
)

simulation_settings=SimulationSettings(
    equilibration_length=0.1 * unit.nanosecond,
    production_length=0.1 * unit.nanosecond,
    checkpoint_storage=None,
    minimization_steps=10000,
    output_filename='rbfe.nc',
    output_indices='all',
    checkpoint_interval=50,
)
```
```
    integrator_settings=IntegratorSettings(
  File "pydantic/main.py", line 404, in pydantic.main.BaseModel.__init__
  File "pydantic/main.py", line 1040, in pydantic.main.validate_model
  File "pydantic/fields.py", line 737, in pydantic.fields.ModelField.validate
  File "pydantic/fields.py", line 913, in pydantic.fields.ModelField._apply_validators
  File "pydantic/class_validators.py", line 277, in pydantic.class_validators._generic_validator_cls.lambda3
  File "/db2/users/isjoung/openfe/openfe/setup/methods/openmm/equil_rbfe_methods.py", line 332, in is_time
    if not v.is_compatible_with(unit.picosecond):
AttributeError: 'NoneType' object has no attribute 'is_compatible_with'

    simulation_settings=SimulationSettings(
  File "pydantic/main.py", line 406, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for SimulationSettings
checkpoint_interval
  instance of Quantity expected (type=type_error.arbitrary_type; expected_arbitrary_type=Quantity)
```

**Comment by IAlibay** [comment]
Hi @insukjoung, thanks so much for raising this issue!

I was wondering if you could provide some information about what your python environment looks like?

i.e. what is the output of `conda list`

Knowing which versions cause the break will be useful in debugging this.

**Comment by insukjoung** [comment]
It looks like the error is related to https://github.com/pydantic/pydantic/issues/3308. It is good to hear that the fix is on-going. I will look forward to #142.

My openfe is git up-to-date version. Conda python is 3.8.10.
openfe                    0.3.post160+gc451ed4           dev_0    <develop>
pydantic                  1.8.2            py38h497a2fe_2    conda-forge

**Comment by richardjgowers** [comment]
@insukjoung This got fixed in the 0.4 release, I'd invite you try again
